apps/client/src/pages/home/Home.page.tsx

```diff
@@ -13,20 +14,31 @@ import {
   ButtonWithInput,
   Description,
   Input,
+  InputState,
   Link,
   Panel,
+  Text,
 } from '../../components';
+import { TextSize, TextType } from '../../models';
 import { endSocket, initializeSocket } from '../../services/initialize-socket';
 
 export function Home(): ReactElement {
   const { roomId } = useParams();
-  // disconnect any socket connection if user navigates to home page.
+  // disconnect any socket connection if user navigates back to home page.
   endSocket();
```

Comment:
This was changed from an upcoming PR #73  66ea107312d0f51fe1fb478601b601bc0b570d45